test/test_common/network_utility.cc

```diff
@@ -116,10 +116,25 @@ Address::InstanceConstSharedPtr getCanonicalLoopbackAddress(Address::IpVersion v
   return Network::Utility::getIpv6LoopbackAddress();
 }
 
-Address::InstanceConstSharedPtr getAnyAddress(const Address::IpVersion version) {
+// There is no portable way to initialize sockaddr_in6 with a static initializer, do it with a
+// helper function instead
+static sockaddr_in6& v6any_() {
+  static sockaddr_in6 v6any = {};
```

Comment:
What do you think about dropping the static modifier for this variable? (Since you rely on the function only being called once to initialize the static variable in getAnyAddress.)

Reply:
Done, had to change the function to return the struct to not return a reference to the stack.